unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/Tests/FriendsControllerShould.cs

```diff
@@ -41,21 +42,18 @@ public void Initialize()
         [Test]
         public void AddFriends()
         {
-            var updatedFriends = new Dictionary<string, FriendshipAction>();
-            var totalFriends = 0;
-            controller.OnUpdateFriendship += (s, action) => updatedFriends[s] = action;
-            controller.OnTotalFriendsUpdated += i => totalFriends = i;
-
-            apiBridge.OnFriendsAdded += Raise.Event<Action<AddFriendsPayload>>(
+            _ = apiBridge.GetFriendsAsync(Arg.Any<int>(), Arg.Any<int>(), Arg.Any<CancellationToken>()).Returns(UniTask.FromResult(
```

Comment:
no need for `_ =` right?

Reply:
It is not really needed but the IDE suggest you to do it in a warning.